syncstorage/storage/sql/queries_spanner.py

```diff
@@ -89,8 +89,10 @@ def FIND_ITEMS(bso, params):
         query = query.where(bso.c.id.in_(params.get("ids")))
     if "newer" in params:
         query = query.where(bso.c.modified > bindparam("newer"))
+        query = query.where(bso.c.modified <= text("CURRENT_TIMESTAMP()"))
```

Comment:
I can imagine future readers wondering why this is here, could you please add a comment to explain the rationale?

Reply:
Or link to relevant Spanner docs?